Hello World/css/style.css

```diff
@@ -0,0 +1,4 @@
+/* Pagraph Color */
+body{
```

Comment:
Good use of CSS body, small comment. It's better to leave a blank space after the selector name, to improve readability.